HW8(MVC)/scripts/View.js

```diff
@@ -0,0 +1,131 @@
+import { $C } from './Controller.js';
+
+class View {
+  constructor() {
+    this.toDoContainer = document.querySelector('.to-do');
+
+    const h1 = document.createElement('h1');
+    h1.innerText = 'THINGS TO DO ✌️';
+    document.body.prepend(h1);
+
+    this.showButton = document.createElement('button');
+    this.showButton.classList.add('to-do__button', 'to-do__button--show');
+    this.showButton.title = 'Show all tasks';
+    this.showButton.innerText = 'SHOW';
+    this.toDoContainer.appendChild(this.showButton);
+
+    this.input = document.createElement('input');
+    this.input.classList.add('to-do__input');
+    this.input.placeholder = 'Add a new task 👈';
+    this.toDoContainer.appendChild(this.input);
+
+    this.addButton = document.createElement('button');
+    this.addButton.classList.add('to-do__button', 'to-do__button--add');
+    this.addButton.title = 'Add a new task';
+    this.addButton.innerText = 'ADD';
+    this.toDoContainer.appendChild(this.addButton);
+
+    this.tasksContainer = document.createElement('div');
+    this.tasksContainer.classList.add('tasks');
+    this.toDoContainer.appendChild(this.tasksContainer);
+
+    this.tasksList = document.createElement('ul');
+    this.tasksList.classList.add('tasks__list');
+    this.tasksContainer.appendChild(this.tasksList);
+  }
+
+  renderTasks(tasks) {
+    if (tasks.length === 0 && $C.isFiltered() === 'false') {
```

Comment:
Why would you take a controller as an unobvious dependency? 
the method can only work with input parameters